README.md

```diff
@@ -31,37 +31,43 @@ Colleagues may collaborate on a spreadsheet that looks like this:
+operations as well as some field format adjustments. It's up to you to integrate them
 in a complete pipeline that ingests and outputs the data however you need.
 
-Scroll down to the [Examples](#examples) section below, to get right to a
+Scroll down to the [Examples](#examples) section below to get right to a
 concrete example you can play with.
 
 ## Maturity
 
 This code is a proof of concept and is not officially supported.
-The pipelines generated by this tool are not meant to be complete, nor production-ready.
+The pipelines generated by this tool are likely not complete and probably need more
+testing and validation before they are ready for production.
 They are simply meant to give you a head start in mapping various sources to ECS.
 
 ## CSV Format
 
-Here's more details on the CSV format supported by this tool. Since mapping
-spreadsheets are primarily used by humans, it's totally fine to have as many columns
+Here are more details on the CSV format supported by this tool. Since mapping
+spreadsheets are used by humans, it's totally fine to have as many columns
 as you need in your spreadsheets/CSV. Only the following columns will be considered:
 
 | column name | required | allowed values | notes |
 |-------------|----------|----------------|-------|
 | source\_field | required |  | A dotted Elasticsearch field name. Dots represent JSON nesting. Lines with empty "source\_field" are skipped. |
 | destination\_field | required |  | A dotted Elasticsearch field name. Dots represent JSON nesting. Can be left empty if there's no copy action (just a type conversion). |
-| format\_action | optional | to\_float, to\_integer, to\_string, to\_boolean, to\_array, uppercase, lowercase, (empty) | Simple conversion to apply to the field value. |
+| format\_action | optional | to\_float, to\_integer, to\_string, to\_boolean, to\_array, parse\_timestamp, uppercase, lowercase, (empty) | Simple conversion to apply to the field value. |
+| timestamp\_format | optional | UNIX, UNIX\_MS, ISO8601, TAI64N, or a Java time pattern. |
 | copy\_action | optional | rename, copy, (empty) | What to do with the field. If left empty, default action is based on the `--copy-action` flag. |
 
 You can start from this
 [spreadsheet template](https://docs.google.com/spreadsheets/d/1m5JiOTeZtUueW3VOVqS8bFYqNGEEyp0jAsgO12NFkNM). Make a copy of it in your Google Docs account, or download it as an Excel file.
 
+When the destination field is @timestamp, then we always enforce an explicit date ```format\_action``` of ```parse\_timestamp``` to ```UNIX_MS``` avoid conversion problems downstream. If no ```timestamp\_format``` is provided, then ```UNIX_MS``` is used. Please note that the timestamp layouts used by the [Filebeat processor for converting timestamps](https://www.elastic.co/guide/en/beats/filebeat/current/processor-timestamp.html) are different than the formats supported by date processors in Logstash and Elasticsearch Ingest Node.
```

Comment:
Please remove the backslash when inside the code backticks ([see it rendered](https://github.com/elastic/ecs-mapper/blob/49d22b55599c18d2a9fadf8eaef723f0441a709c/README.md#csv-format)).